src/controllers/user.go

```diff
@@ -2,22 +2,23 @@ package controllers
 
 import (
 	"encoding/json"
-	"github.com/gin-gonic/gin"
 	"log"
 	"minitwit/src/database"
 	"minitwit/src/functions"
 	"minitwit/src/models"
 	"net/http"
+
+	"github.com/gin-gonic/gin"
 )
 
 func userHandlers(router *gin.Engine) {
 
-	router.GET("/:user/follow", func(c *gin.Context) {
+	router.POST("/:user/follow", func(c *gin.Context) {
 		username := c.Param("user")
 		handleFollowUser(c.Writer, c.Request, c, username)
 	})
 
-	router.GET("/:user/unfollow", func(c *gin.Context) {
+	router.POST("/:user/unfollow", func(c *gin.Context) {
```

Comment:
Same as above

Reply:
Was sure I changed this... must have forgotten to commit before creating this pull request